docs/maintaining/tasks.md

```diff
@@ -14,13 +14,16 @@ To build new images and publish them to the Docker Hub registry, do the followin
 
 ## Updating the Ubuntu Base Image
 
-When there's a security fix in the Ubuntu base image or after some time passes, it's a good idea to update the pinned SHA in the
-[jupyter/base-notebook Dockerfile](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile).
-Submit it as a regular PR and go through the build process.
-Expect the build to take a while to complete: every image layer will rebuild.
+Latest LTS Ubuntu docker image is used as the base image for `minimal-notebook`.
```

Comment:
Hmmm??? The base image is used as a base image for the `minimal-notebook`, which in turn builds on the latest ubuntu long-term support (LTS) image.

Reply:
I rewrote this part, thank you.